core/common/src/test/java/alluxio/PropertyKeyTest.java

```diff
@@ -58,6 +72,12 @@ public void isValid() throws Exception {
   }
 
   @Test
+  public void aliasIsValid() throws Exception {
+    assertTrue(PropertyKey.isValid(mTestProperty.toString()));
+    assertTrue(PropertyKey.isValid(mNullAliasTestProperty.toString()));
```

Comment:
- this is good:
```java
assertTrue(PropertyKey.isValid("alluxio.test.property.alias1")); assertTrue(PropertyKey.isValid("alluxio.test.property.alias2")); assertFalse(PropertyKey.isValid(null));
```
- Good to have null check in `isValid` and `fromString`. You can submit separate PRs on it as it looks less related to this PR. thanks